contrib/cirrus/pr-should-link-jira.t

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+#
+# tests for pr-should-link-jira.t
+#
+
+ME=$(basename $0)
+
+# Our test script queries github, for which we need token
+if [[ -z "$GITHUB_TOKEN" ]]; then
+    echo "$ME: Please set \$GITHUB_TOKEN" >&2
+    exit 1
+fi
+export CIRRUS_REPO_CLONE_TOKEN="$GITHUB_TOKEN"
+
+
+###############################################################################
+# BEGIN test cases
+#
+# Feel free to add as needed. Syntax is:
+#    <exit status> <commit message> <pr> # comments
+#
+# Where:
+#    exit status        is the expected exit status of the script
+#    commit message     commit message list
+#    dest branch        name of branch
+#    pr                 pr number
```

Comment:
This needs to be closer to line 64, where the test table is defined

Reply:
done